pkg/resources/validation/cluster/provisioningcluster.go

```diff
@@ -23,6 +23,7 @@ import (
 const globalNamespace = "cattle-global-data"
 
 var mgmtNameRegex = regexp.MustCompile("^c-[a-z0-9]{5}$")
+var fleetNameRegex = regexp.MustCompile("^[^\\-][-a-z0-9]+$")
```

Comment:
nit: I'm not familiar, but is the \\ necessary here?

Reply:
I retested and I think you're right, it's not needed. The behavior of the regex doesn't change when its removed, but I'll take it out just for clarity